extensions/binding/org.eclipse.smarthome.binding.tradfri/ESH-INF/i18n/tradfri_de.properties

```diff
@@ -0,0 +1,29 @@
+# binding
+binding.tradfri.name = TR�DFRI Erweiterung
```

Comment:
Do you really want to translate "Binding" to German? Everywhere where people talk German about ESH, the also use the word "Binding" - I would keep that. "Erweiterung" is rather the equivalent for "Extension", which describes any kind of additional functionality.